README.md

```diff
@@ -2,8 +2,7 @@
 
 <img src="logo/broccoli-logo.generated.png" align="right" height="150">
 
-[![Build Status](https://travis-ci.org/broccolijs/broccoli.svg?branch=master)](https://travis-ci.org/broccolijs/broccoli)
-[![Build status](https://ci.appveyor.com/api/projects/status/jd3ts93gryjeqclf/branch/master?svg=true)](https://ci.appveyor.com/project/joliss/broccoli/branch/master)
+[![Build Status](https://github.com/broccolijs/broccoli/workflows/nodejs/badge.svg)
```

Comment:
When I try to preview, this doesn't seem to be working.

I think you need:

```suggestion
[![Build Status](https://github.com/broccolijs/broccoli/workflows/CI/badge.svg)](https://github.com/broccolijs/broccoli/actions)
```

Main changes:

* Wrap the badge in a link that will open `/actions` when you click it (otherwise it'll open the SVG file, which is definitely not what you want)
* The name field of the `.yml` file is what is used: `CI` (not the filename which was `nodejs`)